Persistence API Idempotency (#2108)
corda/corda-docs-portal: content/en/platform/corda/5.2/developing-applications/api/application/persistence.md

2. To create a `Dog` entity that writes a row to the database, use the following code:
   ```kotlin
     val dog = Dog(dogId, "dog", Instant.now(), "none")
     persistenceService.persist("persist-dog", dog)
    ```

>     `persist-dog` is a unique (within the context of the flow) deterministic deduplication ID to enable request idempotence in failure scenarios. The ID must be deterministic at the time of the function call and can not be more than 128 characters.

Reviewer: We should note that, as the ID must be deterministic, you should _not_ use a random UUID

>     `persist-dog` is a unique (within the context of the flow) deterministic deduplication ID to enable request idempotence in failure scenarios. The ID must be deterministic at the time of the function call and so you can not use a a random UUID. The ID can not be more than 128 characters.

Reviewer 2: ```suggestion
    `persist-dog` is a unique (within the context of the flow) deterministic deduplication ID to enable request idempotence in failure scenarios. The ID must be deterministic at the time of the function call; therefore, you can not use a a random UUID. The ID can not be more than 128 characters.
```
